### DISCUSSION TITLE
While I am implementing swagger I am facing the issue

### DISCUSSION THREAD

**Discussion by SreedharLilly** [jhipster/generator-jhipster#25328] [General]
![image](https://github.com/jhipster/generator-jhipster/assets/161312747/fd432720-5cd0-4f40-a42c-7d25c9b1b672)
<dependency>
<groupId>org.springdoc</groupId>
         <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
	<version>2.0.2</version>
</dependency>
@Bean
  public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
    http.csrf(csrf -> csrf.disable())
        .exceptionHandling(exception -> exception.authenticationEntryPoint(unauthorizedHandler))
        .sessionManagement(session -> session.sessionCreationPolicy(SessionCreationPolicy.STATELESS))
        .authorizeHttpRequests(auth -> 
          auth.requestMatchers("/api/auth/**").permitAll()
              .requestMatchers("/api/test/**").permitAll()
              .requestMatchers(AntPathRequestMatcher.antMatcher("/**")).permitAll()
              .anyRequest().authenticated()
        );

**Comment by SreedharLilly** [comment]
Anyone have any idea please help on this

**Comment by mraible** [comment]
Are you seeing this in a JHipster app? If so, please provide steps to reproduce the problem.
